Testscripts/Linux/validate-da.sh

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env bash
+#######################################################################
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+#
+# validate-da.sh
+# Description:
+#    Validate the installation and uninstallation of the Dependency Agent.
+#    Validate the enable and disable of the Dependency Agent.
+#######################################################################
+
+set -e
+set -x
+
+# Source utils.sh
+. utils.sh || {
+    echo "ERROR: unable to source utils.sh!"
+    echo "TestAborted" > state.txt
+    exit 0
+}
+
+function check_prereqs() {
+	LogMsg "Checking for preconditions"
+
+	if [[ $EUID -ne 0 ]]; then
+	   LogErr "This script must be run as root" 
+	   exit 1
```

Comment:
please always exit 0 in test script and before that set SetTestStateAborted, SetTestStateFailed. then test framework will handle them.